charts/vmclarity/values.yaml

```diff
@@ -0,0 +1,367 @@
+global:
+  imageRegistry: ""
+
+apiserver:
+  # -- Number of replicas for the API Server
+  replicas: 1
+
+  image:
+    # -- API Server image registry
+    registry: ghcr.io
+    # -- API Server image repositiory
+    repository: openclarity/vmclarity-apiserver
+    # -- API Server image tag (immutable tags are recommended)
```

Comment:
> immutable tags are recommended

Not sure if `immutable tags` are actually exist :)

Also, don't we want to put here `digest` option since I saw it in the helper file.